hello-gl2/app/src/main/AndroidManifest.xml

```diff
@@ -4,35 +4,36 @@
 **
 ** Copyright 2009, The Android Open Source Project
 **
-** Licensed under the Apache License, Version 2.0 (the "License"); 
-** you may not use this file except in compliance with the License. 
-** You may obtain a copy of the License at 
+** Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This is the only sample with a copyright in the androidmanifest -- should they all have it, or is this just a one-off weirdo?